src/helpers/dom/element.js

```diff
@@ -413,20 +413,20 @@ var textContextSupport = document.createTextNode('test').textContent ? true : fa
 export function fastInnerText(element, content) {
   var child = element.firstChild;
 
-  if (child && child.nodeType === 3 && child.nextSibling === null) {
+  if (!child || child.nodeType !== 3 || child.nextSibling !== null) {
```

Comment:
Are you sure?
In the previous version all three comparisons executed every time. In the new version if first is false the rest will skip.